### DISCUSSION TITLE
能否实现更新条目信息

### DISCUSSION THREAD

**Discussion by jiadong825** [windingwind/zotero-actions-tags#175] [Q&A] [answered]
有时候加入zotero的条目信息是不完整的 例如摘要没有自动导入，会议论文集标题没有导入等

**Comment by windingwind** [comment, marked as answer]
如果信息来源也没有完整数据，那只能自己手填；
如果信息来源有完整数据但是translator没有正确获取，应向forums.zotero.org请求更新translator

**Reply by jiadong825** [reply]
我是从会议网站导入的论文， 如何查看他是否提供了完整的数据呢？ 例如 [https://www.usenix.org/conference/nsdi22/presentation/yang-juncheng](https://www.usenix.org/conference/nsdi22/presentation/yang-juncheng)。

**Reply by windingwind** [reply]
与插件无关，向官方论坛反馈

**Reply by wwzhang** [reply]
这种情况现在有不少期刊都有，因为文章上网时in press并没有具体期号页码等信息，正式发表才会有。

**Comment by wwzhang** [comment]
看 https://forums.zotero.org/discussion/75796/how-to-retrieve-and-update-metadata-in-zotero

这个代码7也可以用，但在zotero-actions-tags插件里直接运行会报错

**Reply by windingwind** [reply]
https://github.com/windingwind/zotero-actions-tags/discussions/182
